000_introduction/000_introduction.tex

```diff
@@ -1,3 +1,9 @@
+\chapter{Abstract}\label{abstract}
+
+\section{Abstract}
+
+Co-operative survival games refer to a subset of political choice games wherein `players' must work together to overcome disaster, else suffer the consequences through both personal and communal damage. Furthermore, without the ability to monitor a system and in the absence of a centralised authority, it becomes impossible to form enduring self-governing institutions, resulting in the proposed self-organising mechanisms for `solving' such games potentially functioning in harmful ways. This paper utilises a self-organising, multi-agent system to simulate an iterated collective action, co-operative survival game comprising a resource management problem, the `solution' to which involves a system of localised governance. Through a series of experiments where the behaviour of the players in the system is increasingly randomised and the presence of self-organising mechanisms is toggled, we investigate the causes of instability. Ultimately, we conclude that, through the integration of governance, it is possible for such a system to effectively self-organise to minimise the total deaths of the players involved.
```

Comment:
This work can probably not be referred as a paper (I'm not sure which journal would accept a 150 pages long paper hahah). Maybe simply use "project" or "work".

Reply:
ahh nice spot - force of habit haha